feature/session/src/main/res/values/strings.xml

```diff
@@ -4,6 +4,7 @@
     <string name="start_filter">Filter</string>
     <string name="intended_audience_title">Intended audience</string>
     <string name="session_share" translatable="false">Share</string>
+    <string name="session_register" translatable="false">Register</string>
```

Comment:
`save_event_to_calendar` and `Save to Calendar` sound more natural.

Reply:
updated with this issue f4a19f5